models/openacademy_session.py

```diff
@@ -1,12 +1,48 @@
-from odoo import models, fields
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
 
 class Session(models.Model):
     _name = "openacademy.session"
     _description = "This is the description of Session model."
 
     name = fields.Char()
-    start_date = fields.Date()
+    start_date = fields.Date(default=fields.Date.today)
```

Comment:
Consider user's time zone:

```suggestion
    start_date = fields.Date(default=fields.Date.context_today)
```